### ISSUE TITLE
Fresh desk help page 

### ISSUE THREAD

**Issue by shanile** [Neufund/platform-frontend#723]
We have an issue today that client need to login and create an account with freshdesk in order to be able to raise a ticket or watch their ticket history 

We spoke with Freshdesk today and they said that the reason for it due to the fact we missing the SSO which is Single Sign On / Remote Authentication in Freshdesk
This saves the clients the time and effort involved in creating a separate account for your support portal.

Please read this article and let's see if we can do that as this will create a better user interface 

 https://support.freshdesk.com/support/solutions/articles/31166-single-sign-on-remote-authentication-in-freshdesk

**Comment by SergejSoft** [comment]
@shanile we don't support SSO for security reasons and lack of email update feature. So this ticket will be built only after we have a basic email infrastructure. Consider this on hold.
